portfolio/src/main/webapp/script.js

```diff
@@ -12,6 +12,32 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+/* Create a new HTML element */
+function createElementWithParams (tagName, id, classes, events, innerText) {
```

Comment:
For now, it might be better to only use one class and assume it's not null and have at most one event and assume it's an onclick handler, also remove ID entirely. The reason being that we don't want to over complicate things unless we actually need the functionality. This philosophy is also known as [YAGNI (Ya Aren't Gonna Need It) ](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it).